src/Microsoft.ML.Core/Data/ColumnTypeExtensions.cs

```diff
@@ -77,4 +87,20 @@ public static bool SameSizeAndItemType(this ColumnType columnType, ColumnType ot
             return otherVectorType.Size == otherVectorType.Size;
         }
     }
+
+    /// <summary>
+    /// Extension methods related to the KeyType class.
+    /// </summary>
+    [BestFriend]
+    internal static class KeyTypeExtensions
```

Comment:
We should be putting new classes in their own files, with the file name matching the class name. It gets too hard to navigate when files contain random classes.

Reply:
Makes sense! I moved this to a separate file.

---
In reply to: [250648529](https://github.com/dotnet/machinelearning/pull/2146#discussion_r250648529) [](ancestors = 250648529)